File: DropDownList/CustomItemWithMultipleImages/RadListControlCustomItemsVB/Helpers/BussinessObject.vb
Imports System.ComponentModel

''' <summary>
''' The bussiness object.
''' </summary>
Public Class BussinessObject
    Implements INotifyPropertyChanged

#Region "Constants and Fields"

    ''' <summary>
    ''' The image.
    ''' </summary>
    Private m_image As Image

    ''' <summary>
    ''' The image 2.
    ''' </summary>
    Private m_image2 As Image

    ''' <summary>
    ''' The name.
    ''' </summary>
    Private m_name As String

#End Region

#Region "Events"

    ''' <summary>
    ''' The property changed.
    ''' </summary>
    Public Event PropertyChanged(ByVal sender As Object, ByVal e As System.ComponentModel.PropertyChangedEventArgs) _
                Implements INotifyPropertyChanged.PropertyChanged

#End Region

#Region "Properties"

    ''' <summary>
    ''' Gets or sets Image.
    ''' </summary>
    Public Property Image() As Image
        Get
            Return Me.m_image
        End Get

        Set(ByVal value As Image)
            Me.m_image = value
            Me.OnNotifyPropertyChanged("Image")
        End Set
    End Property

    ''' <summary>
    ''' Gets or sets Image2.
    ''' </summary>
    Public Property Image2() As Image
        Get
            Return Me.m_image2
        End Get

        Set(ByVal value As Image)
            Me.m_image2 = value
            Me.OnNotifyPropertyChanged("Image2")
        End Set
    End Property

    ''' <summary>
    ''' Gets or sets Name.
    ''' </summary>
    Public Property Name() As String
        Get
            Return Me.m_name
        End Get

        Set(ByVal value As String)
            If Me.m_name = value Then
                Return
            End If

            Me.m_name = value
            Me.OnNotifyPropertyChanged("Name")
        End Set
    End Property

#End Region

#Region "Methods"

    ''' <summary>
    ''' The on notify property changed.
    ''' </summary>
    ''' <param name="property">
    ''' The property.
    ''' </param>
    Private Sub OnNotifyPropertyChanged(ByVal [property] As String)
        RaiseEvent PropertyChanged(Me, New PropertyChangedEventArgs([property]))
    End Sub

#End Region

End Class

File: DropDownList/CustomItemWithMultipleImages/RadListControlCustomItemsCS/Helpers/BussinessObject.cs
using System.ComponentModel;
using System.Drawing;

namespace RadListControlCustomItems.Helpers
{
    /// <summary>
    /// The bussiness object.
    /// </summary>
    public class BussinessObject : INotifyPropertyChanged
    {
        #region Constants and Fields

        /// <summary>
        /// The image.
        /// </summary>
        private Image image;

        /// <summary>
        /// The image 2.
        /// </summary>
        private Image image2;

        /// <summary>
        /// The name.
        /// </summary>
        private string name;

        #endregion Constants and Fields

        #region Events

        /// <summary>
        /// The property changed.
        /// </summary>
        public event PropertyChangedEventHandler PropertyChanged;

        #endregion Events

        #region Properties

        /// <summary>
        /// Gets or sets Image.
        /// </summary>
        public Image Image
        {
            get
            {
                return this.image;
            }

            set
            {
                if (this.image == value)
                {
                    return;
                }

                this.image = value;
                this.OnNotifyPropertyChanged("Image");
            }
        }

        /// <summary>
        /// Gets or sets Image2.
        /// </summary>
        public Image Image2
        {
            get
            {
                return this.image2;
            }

            set
            {
                if (this.image2 == value)
                {
                    return;
                }

                this.image2 = value;
                this.OnNotifyPropertyChanged("Image2");
            }
        }

        /// <summary>
        /// Gets or sets Name.
        /// </summary>
        public string Name
        {
            get
            {
                return this.name;
            }

            set
            {
                if (this.name == value)
                {
                    return;
                }

                this.name = value;
                this.OnNotifyPropertyChanged("Name");
            }
        }

        #endregion Properties

        #region Methods

        /// <summary>
        /// The on notify property changed.
        /// </summary>
        /// <param name="property">
        /// The property.
        /// </param>
        private void OnNotifyPropertyChanged(string property)
        {
            if (this.PropertyChanged != null)
            {
                this.PropertyChanged(this, new PropertyChangedEventArgs(property));
            }
        }

        #endregion Methods
    }
}